vault/identity_store_util.go

```diff
@@ -1099,7 +1099,20 @@ func (i *IdentityStore) MemDBEntityByID(entityID string, clone bool) (*identity.
 
 	txn := i.db.Txn(false)
 
-	return i.MemDBEntityByIDInTxn(txn, entityID, clone)
+	entity, err := i.MemDBEntityByIDInTxn(txn, entityID, clone)
+	if err != nil {
+		return entity, err
+	}
+	if entity != nil {
```

Comment:
This seems unrelated to my change but is required to get the mount information for the aliases. Otherwise, this function does not return the path and type for each alias, only the accessor.